modules/i18n/src/models/namespaces/console-ns.ts

```diff
@@ -3948,16 +3972,98 @@ export interface ConsoleNS {
                         };
                     };
                     dangerZones: {
-                        terminate: DangerZone;
+                        terminate: {
+                            actionTitle: string;
+                            header: string;
+                            subheader: string;
+                            subheader2?: string;
+                            buttonHint?: string;
+                            buttonDisableHint?: string;
+                        };
                     };
                     notifications: {
-                        getUserSessions: Notification;
-                        terminateAllUserSessions: Notification;
-                        terminateUserSession: Notification;
-                        getAdminUser: Notification;
+                        getUserSessions: {
+                            error?: {
+                            message: string;
+                            description: string;
+                            genericMessage?: string;
+                         }
+                            genericError?: {
+                            message: string;
+                            description: string;
+                            genericMessage?: string;
+                         }
+                            success?: {
+                            message: string;
+                            description: string;
+                            genericMessage?: string;
+                         }
+                         }
+                         ;
+                        terminateAllUserSessions: {
+                            error?: {
+                            message: string;
+                            description: string;
+                            genericMessage?: string;
+                         }
+                            genericError?: {
+                            message: string;
+                            description: string;
+                            genericMessage?: string;
+                         }
+                            success?: {
+                            message: string;
+                            description: string;
+                            genericMessage?: string;
+                         }
+                         }
+                         ;
```

Comment:
Let's nest the sub interfaces to be consistent with the project code style conventions. Please check and format the other places as well.

```suggestion
                        terminateAllUserSessions: {
                            error?: {
                              message: string;
                              description: string;
                              genericMessage?: string;
                            }
                            genericError?: {
                              message: string;
                              description: string;
                              genericMessage?: string;
                            }
                            success?: {
                              message: string;
                              description: string;
                              genericMessage?: string;
                           }
                        };
```

Reply:
Corrected